docs/doc/13-monitor/prometheus-and-granafa.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Prometheus & Grafana
+---
+
+[Prometheus](https://prometheus.io/) is a time series database that features a flexible query language and a multi-dimensional data model.
+
+[Grafana](https://grafana.com/grafana) is an open-source tool used for analyzing and visualizing metrics.
+
+Databend supports using Prometheus to store monitoring and performance metrics, which can be visualized through Grafana.
```

Comment:
This following tutorial guides you through deploying and integrating Databend, Prometheus, and Grafana. In this tutorial, you'll deploy a local Databend and install Prometheus and Grafana with Docker. Before you start, ensure that you have Docker installed.